### PR TITLE
TWIRF4 Workshops

### PR DESCRIPTION
**Pull request by gandro**
Ready to publish. Should me merged *after* the schedule and workshops have been merged on the website :)

image

### CODE REVIEW

**Comment by apiraino** [comment]
Awaiting Kodebox media coverage text

File: _posts/2018-11-02-talks-and-workshops.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - gandro
+---
+
+## Last Batch of Tickets
+
+We have opened our third and last batch of tickets. So get them while they are still available - you’ll be able to purchase your tickets via [🎟 this link 🎟](https://ti.to/asquera-event-ug/rustfest-rome-2018/).
+
+# Talks and Workshops at RustFest Rome
+
+Last week, we finally announced the long awaited talk lineup for Rome on our Twitter accunt [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:
```
Comment (apiraino) [inline _posts/2018-11-02-talks-and-workshops.md]:
```suggestion
Last week, we finally announced the long awaited talk lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:
```

File: _posts/2018-11-02-talks-and-workshops.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+<div style="text-align: center; padding: 1em;">
+  <a href="https://rome.rustfest.eu/schedule/">
+    <strong>📅🎙️📅 Talk Schedule 📅🎙️📅</strong>
+  </a>
+</div>
+
+This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginner and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```
Comment (apiraino) [inline _posts/2018-11-02-talks-and-workshops.md]:
```suggestion
This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```

File: _posts/2018-11-02-talks-and-workshops.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - gandro
+---
+
+## Last Batch of Tickets
+
+We have opened our third and last batch of tickets. So get them while they are still available - you’ll be able to purchase your tickets via [🎟 this link 🎟](https://ti.to/asquera-event-ug/rustfest-rome-2018/).
+
+# Talks and Workshops at RustFest Rome
+
+Last week, we finally announced the long awaited talk lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:
```
Comment (apiraino) [inline _posts/2018-11-02-talks-and-workshops.md]:
```suggestion
Last week, we finally announced the long awaited talks lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peek of the schedule, which is now complete and available on our website:
```

File: _posts/2018-11-02-talks-and-workshops.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "This Week in RustFest 4: Workshops and Talk Schedule"
```
Comment (apiraino) [inline _posts/2018-11-02-talks-and-workshops.md]:
```suggestion
title: "This Week in RustFest 4: Workshops and Talks Schedule"
```

File: _posts/2018-11-02-talks-and-workshops.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Last week, we finally announced the long awaited talk lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peak of the schedule, which is now complete and available on our website:
+
+<div style="text-align: center; padding: 1em;">
+  <a href="https://rome.rustfest.eu/schedule/">
+    <strong>📅🎙️📅 Talk Schedule 📅🎙️📅</strong>
+  </a>
+</div>
+
+This and next week, we are also releasing the workshop program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```
Comment (apiraino) [inline _posts/2018-11-02-talks-and-workshops.md]:
```suggestion
This and next week, we are also releasing the workshops program for Sunday. We are going to have a wide range of workshop topics again this time, both for Rust beginners and experts alike. Follow us on Twitter and watch this spot for future updates regarding workshops:
```

**Comment by gandro** [comment]
Thank you @apiraino :heart:

**Comment by apiraino** [comment]
@gandro Media coverage for Kodebox/Codechain is now stored in the RustFest Rome "Sponsorships" folder for in our gdrive repo. (you may want to edit it a little bit).

**Comment by gandro** [comment]
I reworked this. Should now be ready for publication, anybody who reads is free to merge this.

But please make sure https://github.com/RustFestEU/rome.rustfest.eu/pull/38 is merged berforehands!

File: _posts/2018-11-06-twirf4-talks-and-workshops.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+This Week in RustFest we finalized our schedule for Saturday and Sunday -- and we have are happy to showcase one of our amazing sponsors.
+
+# Last Batch of Tickets
+
+We have opened our third and last batch of tickets. So get them while they are still available -- you’ll be able to purchase your ticket via this <a href="https://ti.to/asquera-event-ug/rustfest-rome-2018/"><strong>🎟 Tickets 🎟</strong></a> link
+
+# Talks and Workshops at RustFest Rome
+
+Last week, we finally announced the long awaited talks lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest) account. We also published a sneak peek of the schedule, which is now complete and available on our website: <a href="https://rome.rustfest.eu/schedule/"><strong>🎙️ Talk Schedule 🎙️</strong></a>
```
Comment (apiraino) [inline _posts/2018-11-06-twirf4-talks-and-workshops.md]:
```suggestion
Last week, we finally announced the long awaited talks lineup for Rome on our Twitter account [@RustFest](https://twitter.com/RustFest). We also published a sneak peek of the schedule, which is now complete and available on our website: <a href="https://rome.rustfest.eu/schedule/"><strong>🎙️ Talk Schedule 🎙️</strong></a>
```

File: _posts/2018-11-06-twirf4-talks-and-workshops.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+RustFest would of course not be RustFest without workshops. We already announced [**🌉 RustBridge**](https://blog.rustfest.eu/rustbridge-rome) as our first workshop a few weeks back. This week we are super excited to announce the remaining workshops we are having on Sunday:
```
Comment (apiraino) [inline _posts/2018-11-06-twirf4-talks-and-workshops.md:19]:
```suggestion
RustFest of course would not be such without workshops. We already announced [**🌉 RustBridge**](https://blog.rustfest.eu/rustbridge-rome) as our first workshop a few weeks back. This week we are super excited to announce the remaining workshops we are having on Sunday:
```

Comment (apiraino) [reply _posts/2018-11-06-twirf4-talks-and-workshops.md:19]: not too sure about that, though

File: _posts/2018-11-06-twirf4-talks-and-workshops.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+  - [**📚 Effective Rust Documentation**](https://rome.rustfest.eu/sessions/effective-rust-documentation) by *Andre 'llogiq' Bogus*
+  - [**📷 GTK and GStreamer**](https://rome.rustfest.eu/sessions/gtk-gstreamer) by *Guillaume Gomez* and *Sebastian Dröge*
+  - [**🌍 Networking & Web**](https://rome.rustfest.eu/sessions/networking-web) by *Hoverbear* and *Yann Leretaille*
```
Comment (apiraino) [inline _posts/2018-11-06-twirf4-talks-and-workshops.md:23]:
```suggestion
  - [**🌍 Networking & Web**](https://rome.rustfest.eu/sessions/networking-web) by *Andrew 'Hoverbear' Hobden* and *Yann Leretaille*
```

Comment (apiraino) [reply _posts/2018-11-06-twirf4-talks-and-workshops.md:23]: maybe?

**Review by apiraino — COMMENTED** [review]
:+1:

File: _posts/2018-11-06-twirf4-talks-and-workshops.md (hunk 7 above)
Comment (gandro) [reply _posts/2018-11-06-twirf4-talks-and-workshops.md:19]: Me neither, I'll have to check the idiom :+1:

File: _posts/2018-11-06-twirf4-talks-and-workshops.md (hunk 8 above)
Comment (gandro) [reply _posts/2018-11-06-twirf4-talks-and-workshops.md:23]: I think they prefer to go by "@Hoverbear" these days!

Edit: Actually, Hoverbear, are you able to give this workshop with travels and all?

**Pull request merged**